lib/features/home_screen/widgets/contact_widget.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/material.dart';
+
+import '../../../utils/context_extensions.dart';
+import 'contact_icon_widget.dart';
+
+class ContactWidget extends StatelessWidget {
+  const ContactWidget({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(children: [
+      Text(context.localize.contact),
+      const SizedBox(height: 15.0),
+      ListView.builder(
+        itemBuilder: (context, index) {
+          return ContactIconWidget(url: "item $index");
+        },
+      )
+    ]);
+  }
+}
+
+const socialMediaIcons = [];
```

Comment:
Hmm, maybe it's better to determine icon type else where and here just paste proper url.  In similar fashion like in toPWr.

For more details pls check `study_circle_details.dart`

Reply:
this list isn't even used? what was the idea here?